python/py-botocore/Portfile

```diff
@@ -17,17 +17,18 @@ homepage            https://github.com/boto/botocore
 master_sites        pypi:b/botocore
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  6e8b19496da996f93a324e53592c1760737b9000 \
-                    sha256  ca3a5bcf0e1504137bc8b4322ae305e1b0e3b85e76480df9f79a4f3b39517788
+checksums           rmd160  dc681fe92f296f5e31789974c9b5597efe1c4cc2 \
+                    sha256  e8e0e462a45c4b34fd1f018564459b739ec0bb7fbb3358a0e2d8f27b7bf7341f
 
 python.versions     27 34
 
 if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools
+    depends_build-append    port:py${python.version}-setuptools
+    depends_lib-append      port:py${python.version}-dateutil \
+                            port:py${python.version}-docutils \
+                            port:py${python.version}-jmespath
 
     test.run            yes
-#    test.target         check
```

Comment:
Oh sorry, I misread the diff. You deleted a line that was _already_ commented out. Got it.